website/src/data/users.tsx

```diff
@@ -180,14 +180,6 @@ const Users: User[] = [
     source: 'https://github.com/algolia/docsearch-website',
     tags: ['favorite', 'opensource', 'product'],
   },
-  {
-    title: 'Amphora Data',
```

Comment:
FWIW, I'd rather not remove sites that just happen to be down unless for good reason. At least we do have the repo available. Could you file an issue in their repo asking about their site's status, revert the removal here, and just add a `// TODO: site unavailable on 12/31/2021`?

Reply:
Sure, gimmie a sec.

Reply:
Done